akka-actor-typed-tests/src/test/scala/docs/akka/typed/IntroSpec.scala

```diff
@@ -50,7 +50,7 @@ object IntroSpec {
         context.log.info("Greeting {} for {}", n, message.whom)
   //#fiddle_code
   //#hello-world-bot
-        println(s"Greeting ${n} for ${message.whom}")
+        context.log.info(s"Greeting {} for {}", n, message.whom)
```

Comment:
what is used for the `fiddle_code` now?
I think it has to bee println there

by the way, maybe this was already correct?